### PR TITLE
feat: perf

### PR DESCRIPTION
**Pull request by loks0n**
Fix some performance issues

### CODE REVIEW

File: src/routes/projects/[projectId]/index.tsx [hunk 1]
```diff
@@ -116,10 +115,7 @@ export default component$(() => {
     );
   }
 
-  const html = marked(project.description, {
-    mangle: false,
-    headerIds: false,
-  });
+  const html = marked(project.description);
```
Comment (loks0n) [inline src/routes/projects/[projectId]/index.tsx:118]: These are `false` by default now.

File: adapters/vercel-edge/vite.config.ts [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
         input: ['src/entry.vercel-edge.tsx', '@qwik-city-plan'],
+        external: ['util'],
       },
       outDir: '.vercel/output/functions/_qwik-city.func',
     },
-    plugins: [vercelEdgeAdapter()],
+    ssr: {
+      noExternal: true,
+    },
+    plugins: [vercelEdgeAdapter(), viteNodePolyfills()],
```
Comment (loks0n) [inline adapters/vercel-edge/vite.config.ts:19]: Vercel Functions does not have `util` module. We have to polyfill it.

File: src/components/layout/Project.tsx [hunk 3]
```diff
@@ -64,7 +62,10 @@ export default component$((props: { project: Project | null }) => {
       </div>
 
       <Link class="object-og" href={`/projects/${project.$id}`}>
-        <img src={AppwriteService.getProjectThumbnail(project.imageId)} />
+        <img
+          src={AppwriteService.getProjectThumbnail(project.imageId)}
+          alt=""
```
Comment (TorstenDittmann) [inline src/components/layout/Project.tsx:67]: is there any reasonable value for the `alt` attribute?

File: src/components/layout/Project.tsx [hunk 4]
```diff
@@ -27,12 +29,8 @@ export default component$((props: { project: Project | null }) => {
...
 
       <div class="object-og" style="height: 200px;">
-        <img class="c-dark-only" src="/images/project-placeholder.png" alt="" />
-        <img
-          class="c-light-only"
-          src="/images/project-placeholder-light.png"
-          alt=""
-        />
+        <img class="c-dark-only" src={ImgProjectPlaceHolder} alt="" />
+        <img class="c-light-only" src={ImgProjectPlaceHolderLight} alt="" />
```
Comment (TorstenDittmann) [inline src/components/layout/Project.tsx:33]: any resonable values here for `alt` attributes?

File: src/components/layout/Project.tsx [hunk 5]
```diff
@@ -27,12 +29,8 @@ export default component$((props: { project: Project | null }) => {
       </div>
 
       <div class="object-og" style="height: 200px;">
```
Comment (TorstenDittmann) [inline src/components/layout/Project.tsx:31]: might make sense to reduce layout shifts, by adding a `width` or `aspect-ratio`

**Review by TorstenDittmann — COMMENTED** [review]
some questions.

Unrelated, we could use https://vercel.com/storage/blob to make the image proxy even faster 👍🏻

**Review by Meldiron — APPROVED** [review]
LGTM. If you have some performance numbers about before and after (build / init load / ...), pls attach to PR.

File: src/components/layout/Project.tsx (hunk 3 above)
Comment (loks0n) [reply src/components/layout/Project.tsx:67]: I don't think so here - we only have tagline, and I didn't want to duplicate it.

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**